docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,294 @@
+
+
+== Iteration: Persistent Storage
+
+As a user of the service I can perform the following actions:
+
+Storage::
+I can request RWX (Read-Write-Many) or RWO (Read-Write-Once) storage from the storage classes which are pre-configured on the cluster
+
+As an engineer of the service I can perform the following actions:
+
+Storage class::
+I can create and customize the storage classes which are pre-configured on the cluster
+
+
+== Iteration: Login (Authentication)
+
+As a user of the service I can perform the following actions:
+
+Cluster login::
+I can log in to my cluster through a user-friendly interface 
+
+As an engineer of the service I can perform the following actions:
+
+Cluster login::
+I can log in to my cluster through a user-friendly interface 
+
+
+== Iteration: Cluster Role-Based Access Control (RBAC)
+
+As an engineer of the service I can perform the following actions:
+
+Role definition::
+I can define roles that specify the permissions a user or group of users should have within their cluster, including the ability to create, update, and delete resources.
+
+Role binding::
+I can bind roles to users or groups of users, effectively granting them the specified permissions.
+
+Role management::
+I can manage my roles and role bindings using Kubernetes APIs, allowing me to dynamically control access to their cluster and its resources.
+
+Cluster-level RBAC::
+I can enforce Cluster RBAC at the cluster level, ensuring that all users and groups have the appropriate permissions and access to their cluster and its resources.
+
+Namespace-level RBAC::
+I can enforce RBAC at the namespace level, allowing me to define different access policies for different parts of their cluster.
+
+
+== Iteration: Container Network Interface (CNI)
+
+As an engineer of the service I can perform the following actions:
+
+Network plugin selection::
+I can choose from a variety of VSHN supported CNI plugins
+
+Network customization::
+I can customize the network configuration to meet the specific needs of the user's workloads and applications, such as specifying network segmentation, IP address ranges, and other network-level attributes.
+
+Network security::
+I can use CNI plugins to enforce network security policies, such as firewalls, network segmentation, and network access controls, helping me to ensure the security of the user's applications and data.
+
+Network performance::
+I can use CNI plugins to optimize network performance for the user's workloads and applications
+
+
+== Iteration: LoadBalancer
+
+As an engineer of the service I can perform the following actions:
+
+Load balancing setup::
+I can set up load balancing for the applications and services, using the Ingress-Controller. This allows me to define how incoming traffic to the applications should be routed and balanced, ensuring that the users have a seamless and reliable experience.
```

Comment:
Are we talking about the Service type `LoadBalancer` in this section? If you, what role does the Ingress controller play here?